src/test/java/io/debezium/connector/vitess/VitessConnectorIT.java

```diff
@@ -1025,6 +1028,152 @@ public void shouldUseSameTransactionIdWhenMultiGrpcResponses() throws Exception
         Testing.Print.enable();
     }
 
+    @Test
```

Comment:
Would it be possible to add a test to verify the behaviour when `EXCLUDE_EMPTY_SHARDS` is not set?

Reply:
If `EXCLUDE_EMPTY_SHARDS` is not set, and we try to stream from the keyspace with an empty shard, it fails and brings down the test vitess instance, so subsequent unrelated tests would also fail. All other tests in the integration tests do not set this variable and it defaults to false so they are effectively testing it without EXCLUDE_EMPTY_SHARDS being set.